src/iOS.Essentials/BasicViewExtensions/UIButtonExtensions.cs

```diff
@@ -55,4 +55,12 @@ public static class UIButtonExtensions
         button.Menu = menu;
         return button;
     }
+
+    [SupportedOSPlatform("ios14.0")]
+    public static T MakeMenuButton<T>(this T button, Func<UIMenu> menu) where T : UIButton
+    {
+        button.ShowsMenuAsPrimaryAction = true;
```

Comment:
Why wouldn't we invoke MakeMenuButton(button, menu: default) instead of setting up same properties?

Reply:
Because we need to create menu every time when button clicked. So use MakeMenuButton(button, menu: default) is useless there (use a method for setting up one property?)

Reply:
What if there'll be two properties in the future?

Reply:
It will be our future problems.
I think that using method that checks parameter for null, then creates and sets button menu and set needed property is overhead here

Reply:
I just got an idea that we should change MakeMenuButton(UIButton button, UIMenu? menu) method.
Menu won't show if UIMenu is null, why does the method allow nullable menu then?

Reply:
@Leo506 Take a good look at the methods.

Reply:
Oh, I understand what do you mean